components/organisms/Layout.js

```diff
@@ -1,19 +1,19 @@
 import PropTypes from "prop-types";
-import Head from "next/head";
 import { ActionButton } from "../atoms/ActionButton";
 import { SearchBar } from "../atoms/SearchBar";
 import { Banner } from "../atoms/Banner";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { useI18n } from "next-rosetta";
 
 /**
  * Component which defines the layout of the page for all screen sizes
  */
-export const Layout = ({
-  bannerText,
-  bannerTitle,
-  children,
-  bannerStartTestingText,
-  bannerBecomeTesterText,
-}) => {
+export const Layout = ({ bannerText, bannerTitle, children }) => {
+  const { locale, locales, asPath } = useRouter(); // Get current locale and locale list
```

Comment:
since we're gonna be using this for all pages including dynamic ones. I think its probably better if the language is passed in as a prop. Static routes ( such as the home page ect ) may use /fr to differentiate between French and English pages but this may not be the case for dynamic pages

Reply:
language is now passed into Layout as prop